bot.go

```diff
@@ -407,11 +407,9 @@ func issuesHandler(w http.ResponseWriter, r *http.Request) {
 	lcBody := strings.ToLower(*payload.Issue.Body)
 	lcTitle := strings.ToLower(*payload.Issue.Title)
 
+	enhancementRegexp = regexp.MustCompile("feature.?request|enhancement")
```

Comment:
It is customary to initialize regexps outside of functions so that they’re only compiled once.